awx/ui_next/src/components/Lookup/MultiCredentialsLookup.jsx

```diff
@@ -71,6 +71,13 @@ function MultiCredentialsLookup(props) {
         loadCredentials(params, selectedType.id),
         CredentialsAPI.readOptions(),
       ]);
+
+      results.map(result => {
+        if (result.kind === 'vault')
+          result.label = `${result.name} | ${result.inputs.vault_id}`;
```

Comment:
ditto https://github.com/ansible/awx/pull/8738/files#r535449988 (and anywhere else we build the name)